src/components/features/ProgressTrack/ProgressTrack.tsx

```diff
@@ -265,114 +270,121 @@ export function ProgressTrack(props: ProgressTracksProps) {
           mt={label ? 1 : 0}
           mr={trackType === TrackTypes.SceneChallenge ? 2 : 0}
         >
-          {onValueChange && (
-            <ButtonBase
-              aria-label={"Decrement Track"}
-              onClick={() => {
-                if (onValueChange) {
-                  const newValue = Math.max(
-                    value - getDifficultyStep(difficulty),
-                    0
-                  );
-                  onValueChange(newValue);
-                  if (newValue === value) {
-                    announce(`${label} is already at zero ticks`);
-                  } else {
-                    announce(`Updated ${label} to ${getValueText(newValue)}`);
-                  }
-                }
-              }}
-              focusRipple
-              sx={(theme) => ({
-                backgroundColor:
-                  theme.palette.darkGrey[
-                    theme.palette.mode === "light" ? "main" : "light"
-                  ],
-                color: theme.palette.darkGrey.contrastText,
-                px: 0.5,
-                "&:hover": {
-                  backgroundColor: theme.palette.darkGrey.dark,
-                },
-                borderTopLeftRadius: `${theme.shape.borderRadius}px`,
-                borderBottomLeftRadius: `${theme.shape.borderRadius}px`,
-              })}
-            >
-              <MinusIcon />
-            </ButtonBase>
+          {trackType && !hideRollButton && useSmallRollButton && (
+            <IconButton onClick={handleRollClick} sx={{ mr: 0.5 }}>
+              <DieIcon />
+            </IconButton>
```

Comment:
Can we add either a [tooltip](https://mui.com/material-ui/react-tooltip/) or an aria-label to this so that screen readers know what's going on?